content/enterprise/releases/2022/v202201-2.mdx

```diff
@@ -32,7 +32,7 @@ APPLICATION LEVEL FEATURES:
 1. Container resource usage metrics are now available in Prometheus format.
 1. The private registry UI now displays a warning message for old versions of provider documentation with a link to the latest version. It also includes an **On this Page** outline for provider documentation that lets you navigate more quickly between sections. 
 1. Added an outline to the public provider documentation page
-1. The [new `tfe-admin rotate-encryption-password` command](https://www.terraform.io/enterprise/admin/infrastructure/active-active) rotates the encryption password on active/active installations of Terraform Enterprise.
```

Comment:
Same question. If the link redirects let's leave the old release notes that introduced this feature untouched.